src/components/accordion/accordion.js

```diff
@@ -0,0 +1,75 @@
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import { Typography } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import { Box } from '@mui/system'
+
+const style = {

```

Comment:
Style is too big, move it to separate file.

Reply:
++